diff -Naur wireshark-1.12.1+g01b65bf.orig/debian/control wireshark-1.12.1+g01b65bf/debian/control
--- wireshark-1.12.1+g01b65bf.orig/debian/control	2014-12-13 11:50:57.732502235 +0100
+++ wireshark-1.12.1+g01b65bf/debian/control	2014-12-13 11:51:25.024176931 +0100
@@ -5,18 +5,18 @@
 Standards-Version: 3.9.5
 Build-Depends: libgtk-3-dev,
 # enable Qt 5 related dependencies for Qt 5-enabled build
- qtbase5-dev, qtbase5-dev-tools, qttools5-dev, qttools5-dev-tools,
+# qtbase5-dev, qtbase5-dev-tools, qttools5-dev, qttools5-dev-tools,
 # enable Qt 4 related dependencies for Qt 4-enabled build
-# qt4-qmake, libqt4-dev,
+ qt4-qmake, libqt4-dev,
  libpcap0.8-dev, flex, libz-dev, debhelper (>= 9), po-debconf, libtool,
  python (>= 2.6.6-3~), python-ply, libc-ares-dev, xsltproc,
  docbook-xsl (>= 1.64.1.0-0), docbook-xml, libxml2-utils, libpcre3-dev,
  libcap2-dev [linux-any] | libcap-dev (>= 2.17) [linux-any],
  bison,
 # enable versioned libgnutls28-dev below to have GPLv2+ compatible GMP
- libgnutls28-dev (>= 3.2.14-1~),
+# libgnutls28-dev (>= 3.2.14-1~),
 # enable backports-compatible libgnutls-dev
-# libgnutls-dev,
+ libgnutls-dev,
  libgcrypt-dev, portaudio19-dev, libkrb5-dev, liblua5.2-dev, libsmi2-dev,
  libgeoip-dev, dpkg-dev (>= 1.16.1~), imagemagick, xdg-utils,
  libnl-genl-3-dev [linux-any], libnl-route-3-dev [linux-any], asciidoc, cmake, w3m
--- wireshark-1.12.1+g01b65bf.orig/debian/rules	2014-12-13 12:37:20.871292126 +0100
+++ wireshark-1.12.1+g01b65bf/debian/rules	2014-12-13 12:37:34.603129934 +0100
@@ -19,7 +19,7 @@
 override_dh_auto_configure:
 	dh_auto_configure -- -DENABLE_GUIDES=ON \
 		-DCMAKE_INSTALL_LIBDIR=/usr/lib/$(DEB_HOST_MULTIARCH) \
-	# -DENABLE_QT5=OFF
+	-DENABLE_QT5=OFF
 
 override_dh_auto_build:
 	dh_auto_build
